docs/rules/no-did-update-set-state.md

```diff
@@ -51,7 +51,7 @@ var Hello = React.createClass({
 
 ```js
 ...
-"no-did-update-set-state": [<enabled>, <mode>]
+"no-did-update-set-state": [<enabled>, <modeInFunctions>, <modeViaMethods>]
```

Comment:
It might be a good idea to use an options object for these modes instead of positional arguments.

That would be a breaking change, so perhaps it should be done in a followup PR.